op-challenger/game/fault/trace/asterisc/state_converter.go

```diff
@@ -58,45 +62,64 @@ func (state *VMState) validateState() error {
 
 // parseState parses state from json and goes on state validation
 func parseState(path string) (*VMState, error) {
```

Comment:
We should be able to remove the `parseState` and `LoadVMStateFromFile` functions now. The tests will just need updating to check that the converter is running the witness subcommand rather than testing the parsing of the state itself.

Also worth noting that the cannon witness subcommand was modified to output the required data in JSON format - I suspect asterisc is currently just printing the state hash from its witness subcommand.  Asterisc doesn't have to do it the same way as cannon (don't need to call the subcommand `witness` etc if you don't want), it just needs to work with this state_converter.

Reply:
Got it, I've removed the unnecessary functions and modified the state_converter_test like cannon's tests. 

As for the asterisc's `witness` subcommand, I just updated the behavior of witness subcommand in [this PR](https://github.com/ethereum-optimism/asterisc/pull/82/files#diff-a8371b2335c235ec193a34d31123c80f87e386e8769f769d0333e7bd15764e86) so that its behavior is aligned with cannon's.